.github/workflows/WebAutomatedTests.yml

```diff
@@ -0,0 +1,45 @@
+name: Run automated end-to-end tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    name: "Automated tests"
+    runs-on: ubuntu-latest
+    env:
+      BROWSER: firefox
+
+    steps:
+      - name: "Checking changed files"
+        id: files
+        uses: jitterbit/get-changed-files@v1
+      - name: "Checking-out code"
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: "Defining environment variables"
+        run: |
+          echo "CHANGED_GLOBAL_FILES=$(echo '${{ steps.files.outputs.all }}' | grep -v '^packages')" >> $GITHUB_ENV
+          echo "PARTIAL_BUILD=$(if [ '$CHANGED_GLOBAL_FILES' = '' ]; then echo 'true'; else echo 'false'; fi)" >> $GITHUB_ENV
+          echo "SINCE_ARG=$(if [ '$PARTIAL_BUILD' = 'true' ]; then echo '--since $GITHUB_SHA'; else echo ''; fi)" >> $GITHUB_ENV
```

Comment:
I think `GITHUB_SHA` is a sha of last commit, but we need a hash of base of our branch

Reply:
You are right, now i changed to use github.event.pull_request.base.ref which return master in most of the cases and I am also checking the event to be pull_request